packages/rol/adapters/belos/src/function/ROL_BelosKrylov.hpp

```diff
@@ -55,6 +55,7 @@
 
 #include "BelosSolverFactory.hpp"   
 #include "BelosSolverManager.hpp"   
+#include "BelosMinresSolMgr.hpp"
```

Comment:
What's this for?

Reply:
Changed this to call the registerCustomSolverFactory but needs further discussion. Please see general note below on this call - this can be automatic but then i think we'll have to put the includes back in the SolverFactory header for all the managers.